gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -82,6 +85,20 @@ def unconfigure_httpd_application
       end
 
       #
+      # Kerberos KRB5 File Methods
+      #
+      def enable_kerberos_dns_lookups
+        updated_content = File.readlines(KERBEROS_CONFIG_FILE).collect do |line|
+          if line =~ /#{KERBEROS_DNS_LOOKUP}/
+            line.gsub('false', 'true')
+          else
+            line
+          end
+        end
+        File.open(KERBEROS_CONFIG_FILE, "w") { |f| updated_content.each { |line| f.puts line } }
+      end
```

Comment:
I'd prefer if we were explicit on the configuration file entries we're modifying, also, let's make a copy of the file before we update it, maybe something like:

```
def update_kerberos_dns_lookup(status)
  krb5config = File.read(KERBEROS_CONFIG_FILE)
  krb5config[/(\s*)dns_lookup_kdc(\s*)=(\s*)(.*)/, 4] = status
  krb5config[/(\s*)dns_lookup_realm(\s*)=(\s*)(.*)/, 4] = status
  FileUtils.copy(KERBEROS_CONFIG_FILE, "#{KERBEROS_CONFIG_FILE}.#{timestamp}")
  File.open(KERBEROS_CONFIG_FILE, "w") { |f| f.write(krb5config) }
end
```

Calling it update_kerberos_dns_lookup("true" or "false")